libs/shared/contentful/src/lib/contentful.client.ts

```diff
@@ -1,3 +1,5 @@
+// Temporarily ignore. To be fixed during CMS refactor FXA-XXXX
```

Comment:
nit: Do we have a Jira ticket number for this yet?

Reply:
Not yet, in other comments I mentioned this will be picked up in the next Milestone, which hasn't started planning yet.